### PR TITLE
Improve translation for contribution part in README.zh_cn

### PR DESCRIPTION
**Pull request by MySuperSoul**

### CODE REVIEW

File: README.zh_cn.md [hunk 1]
```diff
@@ -43,12 +43,12 @@
 &nbsp;      | &nbsp;
 :---------- | :-----
 版权         | 公共领域。
-Comanche055 | Part of the source code for Colossus 2A, the Command Module's (CM) Apollo Guidance Computer (AGC) for Apollo 11<br>`Assemble revision 055 of AGC program Comanche by NASA`<br>`2021113-051. 10:28 APR. 1, 1969`
-Luminary099 | Part of the source code for Luminary 1A, the Lunar Module's (LM) Apollo Guidance Computer (AGC) for Apollo 11<br>`Assemble revision 001 of AGC program LYM99 by NASA`<br>`2021112-061. 16:27 JUL. 14, 1969`
+Comanche055 | 阿波罗11号制导计算机(AGC)中的指令模块(CM), Colossus 2A的部分源代码<br>`Assemble revision 055 of AGC program Comanche by NASA`<br>`2021113-051. 10:28 APR. 1, 1969`
```
Comment (liyishuai) [inline README.zh_cn.md]:
```suggestion
Comanche055 | 阿波罗11号制导计算机（AGC）中的指令模块（CM）, Colossus 2A的部分源代码<br>`Assemble revision 055 of AGC program Comanche by NASA`<br>`2021113-051. 10:28 APR. 1, 1969`
```

File: README.zh_cn.md [hunk 2]
```diff
@@ -43,12 +43,12 @@
...
+Luminary099 | 阿波罗11号制导计算机(AGC)中的登月模块(LM), Luminary 1A的部分源代码<br>`Assemble revision 001 of AGC program LYM99 by NASA`<br>`2021112-061. 16:27 JUL. 14, 1969`
```
Comment (liyishuai) [inline README.zh_cn.md]:
```suggestion
Luminary099 | 阿波罗11号制导计算机（AGC）中的登月模块（LM）, Luminary 1A的部分源代码<br>`Assemble revision 001 of AGC program LYM99 by NASA`<br>`2021112-061. 16:27 JUL. 14, 1969`
```

**Comment by wopian** [comment]
Fixed merge conflict, @MySuperSoul can you accept the 2 commit suggestions 👍

**Comment by MySuperSoul** [comment]
> Fixed merge conflict, @MySuperSoul can you accept the 2 commit suggestions 👍

Ohh, yes of course

**Comment by wopian** [comment]
Thanks! 🎉

**Pull request merged**